v21.1/information-schema.md

```diff
@@ -61,6 +63,21 @@ Column | Description
 `role_name` | Name of a role.
 `is_grantable` | `YES` if the grantee has the admin option on the role; `NO` if not.
 
+### character_sets
+
+<span class="version-tag">New in v21.1:</span> `character_sets` identifies the character sets available in the current database.
+
+Column | Description
+-------|-----------
+`character_set_catalog` | Always `NULL` (unsupported by CockroachDB).
+`character_set_schema` | Always `NULL` (unsupported by CockroachDB).
+`character_set_name` | Name of the character set (i.e., the name of the database encoding).
```

Comment:
I believe this is always "UTF8", not sure if this is worth mentioning though.
https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/information_schema.go